core/internal/monitor/monitor.go

```diff
@@ -413,8 +418,11 @@
 
 			metrics, err := resource.Sample()
 			if err != nil {
-				sm.logger.CaptureError(fmt.Errorf("monitor: error sampling metrics: %v", err))
-				continue
+				if shouldCaptureSamplingError(err) {
+					sm.logger.CaptureError(fmt.Errorf("monitor: error sampling metrics: %v", err))
+				} else {
+					sm.logger.Debug(fmt.Sprintf("monitor: benign sampling error: %v", err))
```

Comment:
Maybe Warn is more appropriate? Would these spam someone's `debug-internal.log` or are they pretty rare?

Reply:
There's usually nothing the user can do about these errors, and, yes, they could potentially spam the debug log.